frontend/src/views/SubmissionView.vue

```diff
@@ -36,6 +40,10 @@ const activeTab: Ref<number> = ref(Number(props.initialTab));
 const loading: Ref<boolean> = ref(true);
 const noteModalVisible: Ref<boolean> = ref(false);
 const permitModalVisible: Ref<boolean> = ref(false);
+const gridView = ref(false);
+const searchTag = ref('');
+const sortOrder: Ref<number> = ref(Number(-1));
+const sortType: Ref<string> = ref('createdAt');
```

Comment:
Make a local object to reference the different sort types. eg. SORT_CREATED_AT, SORT_USER. Remove magic strings that way.